src/components/html-text-input.js

```diff
@@ -23,10 +23,15 @@ export default class HTMLInputView extends React.Component<PropsType, StateType>
 	state = {
 		html: '',
 	}
+	isIOS: boolean = Platform.OS === 'ios';
```

Comment:
Maybe we can use this import `import { isAppleOS } from './platform';`
from gutenberg (file : [platform](https://github.com/WordPress/gutenberg/blob/master/packages/keycodes/src/platform.native.js)) so if something changes in the future we will have better control of that part of the code.

Reply:
Good catch!
It looks like that specific implementation is an internal of `@wordpress/keycodes`, but still it's a good idea to abstract it in the same way in our codebase.

Let's do that and replace all the instances of `Platform.OS === 'ios'` on a separate PR.

Reply:
I agree @etoledom